charts/model-engine/templates/inference_framework_config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: model-engine-inference-framework-latest-config
+  labels:
+    product: common
+    team: infra
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
```

Comment:
should this be just `pre-install`? Yunfeng told me the idea behind this is to decouple upgrading inference framework versions and installs of model-engine, and if we do `pre-upgrade` I think that updates on installing a new version of model-engine

Reply:
also would it be possible to mention why we're putting this in a new configmap?